assets/js/components/Work/work.gql.mock.js

```diff
@@ -6,13 +6,18 @@ export const mockWork = {
   accessionNumber: "Donohue_001",
   administrativeMetadata: {
     preservationLevel: { id: 1, label: "Level 1" },
-    projectCycle: null,
-    projectDesc: [],
-    projectManager: [],
-    projectName: [],
-    projectProposer: [],
-    projectTaskNumber: [],
-    status: null,
+    projectCycle: "Project Cycle Name",
+    projectDesc: ["New Project Description", "Another Project Description"],
+    projectManager: ["New Project Manager", "Another Project Manager"],
+    projectName: ["New Project Name", "Another project name"],
+    projectProposer: ["New Project Proposer", "Another Project Proposer"],
+    projectTaskNumber: ["New Project Task Number", "Another Project Task"],
+    status: {
+      id: "STARTED",
+      label: "Started",
+      scheme: "STATUS",
+    },
+    visibility: mockVisibility,
   },
```

Comment:
Excellent.  Thanks for keeping these in alphabetical order.  It's so much faster to scan the code.